osier/models/dispatch.py

```diff
@@ -78,30 +107,144 @@ class DispatchModel():
     model : :class:`pyomo.environ.ConcreteModel`
         The :mod:`pyomo` model class that converts python code into a
         set of linear equations.
-    upperbound : float
+    objective : float
+        The result of the model's objective function. Only instantiated
+        after :meth:`DispatchModel.solve()` is called.
+    n_timesteps : int
+        The number of timesteps in the model.
+    upper_bound : float
         The upper bound for all decision variables. Chosen to be equal
         to the maximum capacity of all technologies in :attr:`tech_set`.
+    storage_upper_bound : float
+        The upper bound for storage decision variables.
+    penalty : float
+        The penalty applied to the objective function to eliminate
+        simultaneous charging and discharging. Users may need to tune
+        this parameter. Default is 1.0.
+
+    model_initialized : boolean
```

Comment:
```suggestion
    model_initialized : bool
```
I think this will let it be formatted like `int` and `float`.